zephyr/src/supp_wrapper.c

```diff
@@ -0,0 +1,583 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/*
+ * WPA Supplicant / main() function for Zephyr OS
+ * Copyright (c) 2003-2013, Jouni Malinen <j@w1.fi>
+ *
+ * This software may be distributed under the terms of the BSD license.
+ * See README for more details.
+ */
+
+#include <logging/log.h>
+
+#include "supp_main.h"
+#include "includes.h"
+#include "common.h"
+#include "common/defs.h"
+#include "wpa_supplicant/config.h"
+#include "wpa_supplicant_i.h"
+#include "driver_i.h"
+#include "fst/fst.h"
+#include "p2p_supplicant.h"
+#include "wpa_supplicant_i.h"
+#include "wifi_mgmt.h"
+
+//#define MAX_SSID_LEN 32
+
+int cli_main (int, const char **);
+extern struct wpa_supplicant *wpa_s_0;
+
+int zephyr_supp_disable_network(int id)
```

Comment:
@krish2718 From the application developer perspective this is the public API for supplicant, right? If so aren't we then missing an include file?

Reply:
This is still WIP, as you can see this is not called anywhere, so, will remove this for now add it back once its complete.

Reply:
@krish2718 I see now that I wasn't clear in my question. What I really ment was most functions in this file rather than this particular one.

Reply:
Got it, this file is now removed, and will be re-worked later.